PendingReleaseNotes

```diff
@@ -50,6 +50,12 @@
   recommend that users with versioned buckets, especially those that existed
   on prior releases, use these new tools to check whether their buckets are
   affected and to clean them up accordingly.
+* cephfs: Two FS names can now be swapped, optionally along with their IDs,
+  using "ceph fs swap" command. This commands also updates application tag on
+  all the pools of both the FSs. Mirroring should be disabled for both the FSs
+  (because the cephfs-mirror daemon uses fscid internally and changing it
+  while the daemon is running could result in undefined behaviour) and CephX
+  IDs needs to be reauthorized.
```

Comment:
Re-authorization is not need if the user is fine with data of two CephFSs swapped. Otherwise, CephX IDs will need to be upgraded.

I am changing this line to `CephX IDs may need to be reauthorized and CephFSs needs to be re-mounted.`. Let me know what you think.